src/java.management/share/native/libmanagement/VMManagementImpl.c

```diff
@@ -64,7 +60,7 @@ Java_sun_management_VMManagementImpl_initOptionalSupportFields
   (JNIEnv *env, jclass cls)
 {
     jmmOptionalSupport mos;
-    jint ret = jmm_interface->GetOptionalSupport(env, &mos);
+    jmm_interface->GetOptionalSupport(env, &mos);
```

Comment:
Is it worth making any change here?

We currently ignore the return value from GetOptionalSupport, and no doubt have done for years.
So is the fix to just not record the return value, or should we check it?

Making a change to not capture the return value looks like a statement that it should never be checked.  Even if GetOptionalSupport "can't" really fail with the current implementation, that doesn't seem like the right hint to leave.

Other usage in Java_com_sun_management_internal_DiagnosticCommandImpl_getDiagnosticCommandInfo
also does:
 jint ret = jmm_interface_management_ext->GetOptionalSupport(env, &mos);
 
...and also doesn't check the return value.

Reply:
> Is it worth making any change here?

This was needed because I removed
DISABLED_WARNINGS_gcc_VMManagementImpl.c
while changing the makefile.

Reply:
> Other usage in Java_com_sun_management_internal_DiagnosticCommandImpl_getDiagnosticCommandInfo also does: jint ret = jmm_interface_management_ext->GetOptionalSupport(env, &mos);
> 
> ...and also doesn't check the return value.

Seems this one needs the warning disabling because of the unused variable
DISABLED_WARNINGS_gcc_DiagnosticCommandImpl.c := unused-variable